doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,148 @@
+
+Proceed to re-configure high availability on the primary and standby database-only appliances.
+
+=== Re-configuring High Availability on Database-Only Appliances
+
+Re-configure replication on the database-only appliances, and restart services on your cluster.
+
+==== Configuring the Primary Database-Only Appliance
+
+On the primary database-only appliance, initialize the nodes in the database cluster to configure the database replication:
+
+. In the appliance console menu, select *Configure Database Replication*. 
+. Select *Configure Server as Primary*.
+. Set a unique identifier number for the server and enter the database name and credentials:
+.. Select a number to uniquely identify the node in the replication cluster.
+.. Enter the cluster database name.
+.. Enter the cluster database username.
+.. Enter the cluster database password and confirm the password.
+.. Enter the new primary database-only appliance hostname or IP address.
++
+[NOTE]
+====
+The hostname or IP address must be visible to all appliances that communicate with this database, including the {product-title_short} appliances and any global region databases.
+====
++
+.. Confirm that the replication server configuration details are correct, and select `y` to apply the configuration.
+
+
+
+==== Configuring the Standby Database-Only Appliance
+
+The steps to configure the standby database-only appliance are similar to that of the primary database-only appliance, in that they prepare the appliance to be database-only, but as the standby.
+
+On the standby database-only appliance, configure the following:
+
+. In the appliance console menu, select *Configure Database Replication*. 
+. Select *Configure Server as Standby*.
+. Select the database disk. {product-title_short} then activates the configuration.
+. Set a unique identifier number for the standby server and enter the database name and credentials:
+.. Select a number to uniquely identify the node in the replication cluster.
+.. Enter the cluster database name.
+.. Enter the cluster database username.
+.. Enter the cluster database password.
+.. Enter the new primary database-only appliance hostname or IP address.
+.. Enter the new standby database-only appliance hostname or IP address.
++
+[NOTE]
+====
+The hostname or IP address must be visible to all appliances that communicate with this database, including the engine appliances and any global region databases.
+====
++
+.. Select `y` to configure the replication manager for automatic failover.
+.. Confirm that the replication standby server configuration details are correct, and select `y` to apply the configuration. The standby server will then run an initial synchronization with the primary database, and start locally in standby mode.
+. Verify the configuration on the appliance console details screen for the standby server. When configured successfully, *Local Database Server* shows as `running (standby)`. 
+. Start the database service (on both appliances?):
```

Comment:
This will happen as a part of the console process for configuring a standby.

Reply:
Also, I don't think the PG service on the primary should have ever been stopped.